src/App.vue

```diff
@@ -17,9 +17,16 @@ const zindex = ref(0);
 const gameWonModalShown = ref(false); // only show gameWon modal once
 let timer: any;
 
-const darkmode = ref(store.theme === "dark");
+const fullName = ref("");
+const email = ref("");
 
+const darkmode = ref(store.theme === "dark");
+const popupModalShown = ref(
+  !localStorage.getItem("full_name") || !localStorage.getItem("email")
+);
```

Comment:
```suggestion
const fullName = ref("");
const email = ref("");
const popupModalShown = ref(
  !localStorage.getItem("full_name") || !localStorage.getItem("email")
);

const darkmode = ref(store.theme === "dark");
```

Trying to keep darkmode where it was and adding our lines above it, so when we rebase our changes in future, we don't get any weird conflicts.